news/4486.bugfix

```diff
@@ -0,0 +1 @@
+Improve handling of Unicode output from build tools under Python 3.
```

Comment:
Given the discussions, "Improve handling of improperly encoded text output from build tools" would be a more accurate description now.